js/script.js

```diff
@@ -44,17 +57,22 @@ const createTask = (taskTitle) => {
 
 const renderTasks = () => {
   $taskList.innerHTML = "";
-  tasks.forEach(task => {
+  tasks.forEach((task) => {
     const $tasksList = document.createElement("li");
     const $titleElement = document.createElement("span");
     $titleElement.textContent = task.title;
+    if (task.done) {
+      $titleElement.style.textDecoration = "line-through";
+    }
 
     if (task.editMode) {
       const $inputField = document.createElement("input");
       $inputField.type = "text";
       $inputField.value = task.title;
 
-      const $updateButton = createButton("Update", () => updateHandler(task, $inputField.value));
+      const $updateButton = createButton("Update", () =>
+        updateHandler(task, $inputField.value)
+      );
```

Comment:
It's not part of this specific PR I believe

Reply:
Fixed, bhai.